ui/src/lib/helpers.tsx

```diff
@@ -585,3 +585,11 @@ export const htmlToText = (htmlString: string): string => {
         return $(':root').text() || '';
     }
 };
+
+export const toKebabCase = (string: string): string => {
```

Comment:
Might be being pedantic, could you rename string to `inputString` or something. There's too many strings going on here :)

Reply:
Good point! Now changed.